internal/parser/xml_parser.go

```diff
@@ -69,3 +193,28 @@ func convertDptFormat(dpt string) string {
 	}
 	return ""
 }
+
+// translateDatapointTypeToDPTs iterates over all XmlGroupAddress objects nested inside export
+// and checks if DPTs is empty. If so, it sets DPTs = DatapointType
+func translateDatapointTypeToDPTs(export *models.XmlGroupAddressExport) {
```

Comment:
Is this necessary only for when importing from `.knxproj` and why is it that sometimes DPT is nots et but DatapointType is.